### PR TITLE
Feature/header navigation fixes

### PR DESCRIPTION
**Pull request by Mildred14**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -4,15 +4,14 @@
     <title> Finden </title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"> 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (diegosaul402) [inline app/views/layouts/application.html.erb]: I know is not part of your tasks, but please check the extra spaces in this line.

File: app/views/pages/about.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <div class="row">
+    <h1><%= t('about_page.title')%></h1>
```
Comment (diegosaul402) [inline app/views/pages/about.html.erb]: Please check the spaces here.

File: app/views/pages/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <div class="row">
+    <h1><%= t('about_page.title')%></h1>  
+  </div>
+  <div class="row">
+    <div class="col s5">
+      <%= link_to image_tag('about-img.jpg', width: 200, height: 200)%>
```
Comment (diegosaul402) [inline app/views/pages/about.html.erb]: Please check the spaces here.

File: app/views/welcome/_footer.html.erb [hunk 4]
```diff
@@ -7,7 +7,7 @@
         <%= link_to image_tag('mh_logo.png', width: 100, height: 100), class: 'brand-logo' %>
       </div>
       <div class="col l4 offset-l2 s12">
-        <h5 class="white-text"> <%= t('about_us') %> </h5>
+        <h5 class="white-text"> <%= t('footer.about_us') %> </h5>
```
Comment (diegosaul402) [inline app/views/welcome/_footer.html.erb]: The extra spaces before the `<%=` and `</h5>` are required?

File: app/views/welcome/_header.html.erb [hunk 5]
```diff
@@ -1,12 +1,11 @@
 <nav>
-  <div class="nav-wrapper grey darken-4">
-    <%= link_to image_tag('finden.png', width: 150, height: 60), class: 'brand-logo' %>
-    <a href="#!" class="brand-logo"></a>
-    <a href="#" data-target="mobile-demo" class="sidenav-trigger"><i class="material-icons"> <%= t('header.menu') %> </i></a>
+  <div class="nav-wrapper navbar grey darken-4">
+    <%= link_to image_tag("finden.png", alt: "Image Description", class: "logo-nav"), root_path, class:'main-navigation-logo' %>
```
Comment (diegosaul402) [inline app/views/welcome/_header.html.erb]: Please use single quotes when you aren't interpolating strings.

File: app/views/welcome/_header.html.erb [hunk 6]
```diff
@@ -20,8 +19,17 @@
...
-  <li><a href="sass.html">About us</a></li>
-  <li><a href="badges.html">FAQ</a></li>
-  <li><a href="collapsible.html">Contact</a></li>
-  <li><a href="mobile.html">Login</a></li>
+  <li> | <%= link_to t('header.about_us'), pages_about_path %></li>
+  <li><%= link_to t('header.faq'), pages_faq_path %></li>
+  <li><%= link_to t('header.contact'), pages_contact_path %></li>
+  <div class='session-container'>
+    <% if user_signed_in? %>
+      <li>Logged in as <strong><%= current_user.email %></strong>.</li>
```
Comment (diegosaul402) [inline app/views/welcome/_header.html.erb]: Please internationalize this text. You can use interpolation on your strings, you can have this on your locale file 
```
header:
  logged_user: 'Logged in as %{user_email}.'
```
and here you can change this by
```
<li><%= t('header.logged_user', user_email: current_user.email).html_safe %></li>
```

We're doing two things here
- Using variables on the internationalized strings.
- Using HTML tags directly from that text.

File: app/views/welcome/_header.html.erb [hunk 7]
```diff
@@ -20,8 +19,17 @@
...
+      <li><%= link_to t('header.edit_profile'), edit_user_registration_path, class: 'navbar-link' %> |</li>
+      <li><%= link_to t('header.sign_out'), destroy_user_session_path, method: :delete, class: 'navbar-link'  %></li>
+    <% else %>
+      <li><%= link_to t('header.sign_up'), new_user_registration_path, class: 'navbar-link '  %></li>
+      <li><%= link_to t('header.log_in'), new_user_session_path, class: 'navbar-link'  %></li>
```
Comment (diegosaul402) [inline app/views/welcome/_header.html.erb]: Please check the spaces in all these elements too.

**Comment by 8geonirt** [comment]
Awesome job, let's merge this :nerd_face:

**Pull request merged**